checker/tests/nullness/Issue880.java

```diff
@@ -0,0 +1,22 @@
+// Test case for Issue 880:
```

Comment:
So, it's an additional test for storing annotations in extends/implements clause.  We don't have an issue for that.  So maybe we should open one and then add reference that issue along with PR #880 and #876?